docs/initial-setup.md

```diff
@@ -0,0 +1,24 @@
+# Initial setup
```

Comment:
nit: this file got excluded from code review, is .gitattributes over-matching ?

Reply:
should `docs/initial-setup.md linguist-generated=false` do the trick?